java/src/jmri/Sensor.java

```diff
@@ -26,6 +26,26 @@
     public static final int ACTIVE = 0x02;
     public static final int INACTIVE = 0x04;
 
+    // this should only be used for debugging…
+    public String toString() {
```

Comment:
There's a new `describeState(getState())` idiom that does this, just added to the code in the past couple days.  (it's probably not in your repository until you do a "git pull" to get the most recent contents)